modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss

```diff
@@ -42,15 +42,21 @@ svg {
 
 // Style the icons with external links.
 a {
-  // Add the underline to all links.
-  text-decoration: underline;
-
```

Comment:
Are you sure you want to remove the underline from non-external links?

Reply:
@sherakama, yes, this was getting overridden. I move it down to line 47 to be more specific.

Reply:
I think we do want to continue to have the underline on all link tags regardless of destination. Let's keep it here.